src/Stratis.Bitcoin.IntegrationTests/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -53,12 +54,11 @@ public CoreNode(string folder, INodeRunner runner, NodeBuilder builder, Network
             this.runner = runner;
             this.Folder = folder;
             this.State = CoreNodeState.Stopped;
-            if (cleanfolders)
-                this.CleanFolder();
 
-            Directory.CreateDirectory(folder);
             this.DataFolder = Path.Combine(folder, "data");
```

Comment:
not sure why the DataFolder is even set manually. We should create a node using -datadir and Datafolder should be set automatically.